app.js

```diff
@@ -21,7 +21,9 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/', userUUIDMiddleware, require('./routes/index'));
+app.use(userUUIDMiddleware);
+
+app.use('/', require('./routes/index'));
```

Comment:
why not use s3o on the index here, rather than on every route in index.js?
reorder the routes so that it only applies to `/` ?